addon/components/nrg-datetime.hbs

```diff
@@ -0,0 +1,49 @@
+<div
+  class="ui calendar"
+  role="button"
+  {{on "click" this.onFocus}}
+  {{!-- {{on-click-outside this.onBlur}} --}}
+>
+  {{#if hasBlock}}
```

Comment:
[This is deprecated](https://deprecations.emberjs.com/v3.x#toc_has-block-and-has-block-params) to support named blocks:
```suggestion
  {{#if (has-block)}}
```